components/dropdown/dropdown-opener-mixin.js

```diff
@@ -325,10 +287,11 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 	_onOutsideClick(e) {
 		if (!this._isOpen) return;
```

Comment:
Added a check to ensure clicks coming from the opener do not trigger a close - had to add this due to the binding function above. Just a note on performance - I've used the composedAncestor functions many times in this function, BUT they will only be called with the hover option and only for open dropdowns.